client/components/trip/createTripModal.tsx

```diff
@@ -93,8 +93,17 @@ export const SaveTripContainer = ({ dateRange }) => {
   // const [startDate, setStartDate] = useState("");
   // const [endDate, setEndDate] = useState("");
   const [isPublic, setIsPublic] = useState(true);
-
-  // create trip
+  const usePhotonDetailsQuery = () => {
```

Comment:
We should be doing this in the hooks directory not inline here

Reply:
Also we probably don't need to refetch this, i believe its done in the parent component